README.md

```diff
@@ -31,6 +31,26 @@ Then **update the** `.env` file with the correct values.
 
 **Important Note:** Due to the current tiled configuration, ensure that the `WRITE_DIR` is a subdirectory of the `READ_DIR` if the same tiled server is used for both reading data and writing results.
 
+#### MLFlow Configuration in .env
+
+When setting `MLFLOW_TRACKING_URI` in the `.env` file:
+
+- If you run the [MLFlow server](https://github.com/xiaoyachong/mlex_mlflow) locally, you can set it to:
```

Comment:
We should add mlflow to `docker-compose.yml` in this repo to avoid cloning an extra repo